ui/src/literature/components/AssignLiteratureItemDrawer.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useCallback, useState } from 'react';
+import { SelectOutlined } from '@ant-design/icons';
+
+import { Drawer, Radio, Row, Col, Button } from 'antd';
+import ResultItem from '../../common/components/ResultItem';
+import Author from '../../common/components/Author';
+
+const renderAuthorItem = (authors: any, onChange: (event: any) => void) => (
```

Comment:
since we are adding types, `any` it's not a good option cause it makes no difference :)